.github/workflows/build_registry.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         working-directory: ${{ github.workspace }}
 
       - name: Merge Source configs and write Source-Registry
-        run: python .github/build_registry.py
+        run: python .github/build_registry.py .github/registry.yml
```

Comment:
I have added this arg to be able to store it in a different location when built in the workflow